README.md

```diff
@@ -40,9 +42,9 @@ You would have to change a few things (see below, build from scratch)
 
 To run this sample, you'll need:
 
-- [Visual Studio 2017](https://aka.ms/vsdownload)
+- [Visual Studio 2019](https://aka.ms/vsdownload)

```

Comment:
Visual Studio with no version maybe ?

Reply:
Shama, work with Tiago to update the Code generator

---
In reply to: [429118036](https://github.com/Azure-Samples/active-directory-dotnet-native-uwp-v2/pull/38#discussion_r429118036) [](ancestors = 429118036)

Reply:
@jmprieur .net core 3 samples require VS2019. Should we keep the version to avoid people not being able to run the project due to the framework version?